cheroot/__main__.py

```diff
@@ -0,0 +1,134 @@
+"""Command line tool for starting a Cheroot instance.
```

Comment:
Let's clarify that it's WSGI.

Also, I think it might be good to also support starting plain HTTP thing accepting `cheroot.server.Gateway` instance or class descendant.